modules/ossm-creating-certificates-for-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,197 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-certificates-for-a-multi-cluster-mesh_{context}"]
+= Creating certificates for a multi-cluster mesh 
+
+This procedure explains how to create root and intermediate certificates for two clusters, named East and West.
+
+[NOTE]
+====
+In this procedure, `cluster1` is the East cluster and `cluster2` is the West cluster. 
+====
+
+.Prerequisites
+
+* You have OpenSSL installed locally.
+
+.Procedure
+
+. Create a root certificate file named `root-ca.conf`:
++
+.Sample root certificate file
+[source,terminal]
+----
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+[ req_dn ]
+O = Istio
+CN = Root CA
+----
+
+. Create a key for the root certificate by running the following command:
++
+[source,terminal]
+----
+$ openssl genrsa -out root-key.pem 4096
+----
+
+. Create the certificate signing request by running the following command:
++
+[source,terminal]
+----
+openssl req -sha256 -new -key root-key.pem \
+  -config root-ca.conf \
+  -out root-cert.csr
+----
+
+. Create a shared root certificate by running the following command:
++
+[source,terminal]
+----
+openssl x509 -req -sha256 -days 3650 \
+  -signkey root-key.pem \
+  -extensions req_ext -extfile root-ca.conf \
+  -in root-cert.csr \
+  -out root-cert.pem
+----
+
+. Create an intermediate certificate for the East cluster by running the following command:
++
+[source,terminal]
+----
+$ openssl genrsa -out east/ca-key.pem 4096
+----
+
+. Create an OpenSSL configuration file named `intermediate.conf` for the intermediate certificate of the East cluster by copying the following sample file and saving it locally:
```

Comment:
Since the west cluster will also have a file called `intermediate.conf`, you should mention the directory in which to create this file:
```suggestion
. Create an OpenSSL configuration file named `intermediate.conf` in the `east/` directory for the intermediate certificate of the East cluster by copying the following sample file and saving it locally:
```

Reply:
done